modules/custom_operations/setup.py

```diff
@@ -0,0 +1,15 @@
+from skbuild import setup
+from skbuild import constants
+
+setup(
+    packages=["ov_tokenizer"],
+    package_dir={"": "user_ie_extensions/tokenizer/python"},
+    cmake_install_dir="user_ie_extensions/tokenizer/python/ov_tokenizer/libs",
+    cmake_args=['-DCUSTOM_OPERATIONS:STRING=tokenizer',
+                '-DBUILD_FAST_TOKENIZERS=OFF']
+)
+
+# When building extension modules `cmake_install_dir` should always be set to the
```

Comment:
do you think we can drop this file and move all configuration to modules/custom_operations/pyproject.toml ?